Source/WebKit/WebProcess/WebPage/mac/WKAccessibilityWebPageObjectMac.mm

```diff
@@ -340,8 +340,13 @@ - (id)accessibilityHitTest:(NSPoint)point
         if (protectedSelf->m_page->mainFramePlugIn())
             return convertedPoint;
 
-        if (auto* frameView = protectedSelf->m_page->localMainFrameView())
-            convertedPoint.moveBy(frameView->scrollPosition());
+        if (auto* localFrameView = protectedSelf->m_page->localMainFrameView()) {
```

Comment:
I think we need a smart pointer instead of a raw pointer here (I know you only changed the variable name, but good to fix while we're here)